tools/experiments/experiments-config.js

```diff
@@ -315,4 +315,9 @@ export const EXPERIMENTS = [
     spec: 'https://github.com/ampproject/amphtml/issues/24168',
     cleanupIssue: 'https://github.com/ampproject/amphtml/issues/24169',
   },
+  {
+    id: 'amp-sidebar-disable-history-ios',
```

Comment:
Is there any spec thread for this?

Reply:
Also, as I'm looking at this file, what's the benefit of it being JS instead of JSON?